gtfs/importers/gtfs_feed_updater.py

```diff
@@ -17,11 +17,13 @@ def __init__(self, logger=None):
         self.reader = GTFSFeedReader()
 
     def update_feeds(self, force: bool = False):
+        self.logger.info("Importing feeds...")
         for feed in Feed.objects.all():
             try:
                 self.update_single_feed(feed, force=force)
```

Comment:
Is it important to know which feeds got updated?

Reply:
More details about the feed will be logged deeper in the code. This especially for the cases where there are no feeds and we still want to make sure that this function is being ran by the cronjob.